src/language/tekster.ts

```diff
@@ -4,6 +4,10 @@ export default {
     'skjema.': '',
     'skjema.tittel.omarbeidssøker': 'Om deg som arbeidssøker',
 
+    'skjema.oppsummering.disclaimer':
```

Comment:
Trenger vi tilsvarende tekst for engelsk/nynorsk også?

Reply:
Yup, men er mange av de som vi ikke har lagt til ennå. Litt fordi vi ikke helt hadde bestemt oss for "tekstid formatet" 🙈 😁